src/components/nodes/sequenceFlow/sequenceFlowConfig.js

```diff
@@ -1 +1,2 @@
-export const expressionPosition = { offset: -25 };
+export const namePosition = { offset: { y: -20 } };
+export const expressionPosition =  { offset: { y: 40 } };
```

Comment:
I suppose some fiddling with these values could be the answer to Rob's comment?

Reply:
When testing with Rob, you are able to change the position of the label which caused the label to be off-centered.